feconf.py

```diff
@@ -114,10 +114,10 @@
 MAX_LEARNER_PLAYLIST_ACTIVITY_COUNT = 10
 
 # The minimum number of training samples required for training a classifier.
-MIN_TOTAL_TRAINING_EXAMPLES = 50
+MIN_TOTAL_TRAINING_EXAMPLES = 5
```

Comment:
I see. I don't know how such case should be handled. Normally on backend we swap variables but here it won't be possible. @seanlip do you have any idea for this?